kasa/tests/device_fixtures.py

```diff
@@ -15,7 +15,14 @@
 
 from .fakeprotocol_iot import FakeIotProtocol
 from .fakeprotocol_smart import FakeSmartProtocol
-from .fixtureinfo import FIXTURE_DATA, FixtureInfo, filter_fixtures, idgenerator
+from .fixtureinfo import (
```

Comment:
The cleanups for tests should probably go into their own PR?

Reply:
This was done to enable filtering by component version to test the different versions of the component, i.e. v1 vs greater than v1. I added the `ProtocolFilter` type at the same time so can remove that.

Reply:
`ProtocolFilter` removed